storage-proofs/src/stacked/proof.rs

```diff
@@ -177,8 +185,9 @@ impl<'a, H: 'static + Hasher, G: 'static + Hasher> StackedDrg<'a, H, G> {
                                 let labeled_node = rpc.c_x.get_node_at_layer(layer);
                                 assert!(
                                     proof.verify(&pub_inputs.replica_id, &labeled_node),
-                                    "Invalid encoding proof generated"
+                                    format!("Invalid encoding proof generated at layer {}", layer)
```

Comment:
This will cause `seal_commit` to panic if one of the layer files was corrupted (e.g. its contents were replaced with the string `erin totally rules`) after `seal_pre_commit` returned.

@dignifiedquire - Same question as above - how does dealing with these error cases prioritize?

----

To repro the panic, check out branch `mt-persistence-panic` and run:

```
cargo test --color=always --release --package filecoin-proofs --lib api::tests::test_seal_lifecycle_layer_corruption -- --nocapture --exact --ignored
```